docs/Gemfile.lock

```diff
@@ -0,0 +1,169 @@
+GEM
```

Comment:
Is this intended for commit? Seems like an automatically generated file of some sort.